owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/FileListAdapter.kt

```diff
@@ -250,16 +249,25 @@ class FileListAdapter(
 
                 ViewType.GRID_IMAGE.ordinal -> {
                     val view = holder as GridImageViewHolder
+                    val layoutParams = fileIcon.layoutParams as ViewGroup.MarginLayoutParams
```

Comment:
Why `ViewGroup.MarginLayoutParams`?

Reply:
Because if we do not set `MarginLayoutParams` we cannot access the `setMargins()` method that we need to adjust the margins.